installer/manifests/keptn/templates/api-service.yaml

```diff
@@ -17,6 +17,37 @@ spec:
     spec:
       {{- include "keptn.common.pod-security-context" . | nindent 6 }}
       terminationGracePeriodSeconds: {{ .Values.apiService.gracePeriod | default 60 }}
+      initContainers:
+        - name: wait-for-nats
+          image: busybox:1.35.0
+          env:
+          - name: "ENDPOINT"
+            value: "keptn-nats"
+          command: ['sh', '-c', 'until nslookup $ENDPOINT; do echo waiting for $ENDPOINT; sleep 2; done;']
+        - name: wait-for-keptn-mongo
+          image: busybox:1.35.0
```

Comment:
Please make the busybux image version and the endpoint configurable as values. Furthermore, we should fail after some time ...

Reply:
Can you help with this? I tried adding this to `values.yaml`:

```
initContainers:
  image: "busybox"
  version: "1.35.0"
```
and setting line 28 (above) as `image: "{{ .Values.initContainers.image }}:{{ .Values.initContainers.version }}"` and I get a warning during `helm install`:

```
coalesce.go:160: warning: skipped value for initContainers: Not a table
```

No idea how to build in the initContainer timeout.